### DISCUSSION TITLE
Cant get CHAT_ID from channel URL in Telegram Web anymore ?

### DISCUSSION THREAD

**Discussion by ezescigo** [Dineshkarthik/telegram_media_downloader#102] [Q&A] [answered]
I am not able to get the chat_id through the channel's URL since it doesnt change anymore. I am doing anything wrong? is there any other way to get it?

![imagen](https://user-images.githubusercontent.com/60001710/124356667-6f06b080-dc17-11eb-8176-5dbb5bcb7a89.png)

thank you!

**Comment by Dineshkarthik** [comment, marked as answer]
Hi this is due to the new alpha version of the web app, you can switch to the old one like below
![image](https://user-images.githubusercontent.com/7752572/124472530-86bc7100-dd9e-11eb-9936-9527d72152a2.png)

**Comment by ezescigo** [comment]
Thank you! I realised chat_id is the @username so it worked straight away.

Thank you
